app/models/requests/router.rb

```diff
@@ -95,10 +95,12 @@ def calculate_services
               services << 'lewis'
             elsif requestable.recap?
               if requestable.has_item_data?
-                services << 'recap'
+                # No physical recap delivery during campus closure
+                # services << 'recap'
                 if (requestable.recap_edd? && auth_user?)
                   services << 'recap_edd'
                 end
+              # No hysical recap delivery during campus closure
```

Comment:
Guessing this comment line should be removed.  If not remove it add a `P` to `hysical`